setup.py

```diff
@@ -37,6 +37,7 @@
     "flask-talisman>=1.0,<2.0",
     "gevent>=21.12.0,<22.0",
     "gevent-websocket>=0.10.1,<0.11",
+    "taipy-config@git+https://git@github.com/Avaiga/taipy-config.git@develop",
```

Comment:
?

Reply:
You are right. We are getting taipy-config from github (same for taipy-core). We will change this as soon as taipy-config is pushed to pypi